reflectable/lib/capability.dart

```diff
@@ -518,11 +518,11 @@ class _StringInvocation extends StringInvocation {
 /// doesn't have the capabilities to invoke it.
 class ReflectableNoSuchMethodError extends Error
     implements NoSuchCapabilityError {
-  final Object receiver;
+  final Object? receiver;
   final String memberName;
   final List positionalArguments;
-  final Map<Symbol, dynamic> namedArguments;
-  final List<Symbol> existingArgumentNames;
+  final Map<Symbol, dynamic>? namedArguments;
+  final List<Symbol>? existingArgumentNames;
```

Comment:
Ditto here

Reply:
Deleted the field and associated parameters: Turns out that no call sites are able to provide this information, and it was always null.

Clients are not expected to create instances of `ReflectableNoSuchMethodError`, nor are they expected to call `reflectableNoSuch{Method,{G,S}etter,Constructor}Error`. So I jumped to the conclusion that we don't need to make this parameter deprecated, and removed it.